daprdocs/content/en/dotnet-sdk-docs/dotnet-client/_index.md

```diff
@@ -174,12 +174,35 @@ await client.InvokeBindingAsync("send-email", "create", email);
 - For a full guide on output bindings visit [How-To: Use bindings]({{% ref howto-bindings.md %}}).
 
 ### Retrieve secrets
+Prior to retrieving secrets, it's important that the outbound channel be registered and ready or the SDK will be unable
```

Comment:
```suggestion
Prior to retrieving secrets, it's important that the "outbound" channel to Daprd be registered and ready or the SDK will be unable
```

Reply:
Outbound is from daprd to the application, not application to daprd.